core/base/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ android {
 dependencies {
     projects.library.apply {
         api(projects.core.test)
+        api(projects.core.ktor)
```

Comment:
In my opinion we can use something like this:
```
projects.core.network.ktor
```

Reply:
Thank you for sharing your suggestion. That's an good idea, and I agree with you. Using `projects.core.network.ktor` provides a clear and structured naming convention that reflects both the general purpose (network) and the specific implementation (ktor). This approach maintains clarity while allowing for potential future additions or changes. I appreciate your input on this matter.